packages/terra-dropdown-button/translations/fi-FI.json

```diff
@@ -1,4 +1,4 @@
 {
   "Terra.dropdownButton.moreOptions": "More Options",
-  "Terra.dropdownButton.selected": "Valittu."
+  "Terra.dropdownButton.selected": "Valittu"
```

Comment:
we are missing translations here !! Please make sure translations are added from valid file it should be from fi-FI.json folder provided by translation team.

Reply:
I didn't received translations for fi-FI.json when I checked with the translation team she told it is not a supported locale.